implementations/MakoLab.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "MakoLab",
+  "implementation": "MakoLab Verifiable Credentials",
+  "issuers": [
+    {
+      "id": "did:key:zDnaeipTBN8tgRmkjZWaQSBFj4Ub3ywWP6vAsgGET922nkvZz",
+      "endpoint": "https://db.makolab.pro/issuers/credentials/issue",
+      "method": "POST",
```

Comment:
```suggestion
      "method": "POST",
```

Reply:
@BigBlueHat — If `method` is no longer needed, shouldn't your suggestion be --
```suggestion
```

Reply:
Heh...yeah... Reviewing too fast. 🤦‍♂️